src/Akka.HealthCheck.Persistence.Tests/AkkaPersistenceLivenessProbeProviderSettingsTest.cs

```diff
@@ -0,0 +1,36 @@
+// -----------------------------------------------------------------------
+// <copyright file="UnitTest1.cs" company="Petabridge, LLC">
+//      Copyright (C) 2015 - 2019 Petabridge, LLC <https://petabridge.com>
+// </copyright>
+// -----------------------------------------------------------------------
+using Akka.Actor;
+using Akka.Configuration;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Akka.HealthCheck.Persistence.Tests
+{
+    public class AkkaPersistenceLivenessProbeProviderSettingsTest : TestKit.Xunit.TestKit
+    {
+
+        public AkkaPersistenceLivenessProbeProviderSettingsTest(ITestOutputHelper helper)
+            : base(hoconString, output: helper)
+        {
+        }
+        public static string hoconString = @"
```

Comment:
If this is going to be a `public static` member, should follow PascalCase: http://wiki.c2.com/?PascalCase - need to capitalize `h` in `hoconString`.

Reply:
Updated string to follow PascalCase
`public static string HoconString = @"`